Makefile

```diff
@@ -0,0 +1,25 @@
+# kernel-style V=1 build verbosity
+ifeq ("$(origin V)", "command line")
+       BUILD_VERBOSE = $(V)
+endif
+
+ifeq ($(BUILD_VERBOSE),1)
+       Q =
+else
+       Q = @
+endif
+
+.PHONY: all
+all: bin/omaha
```

Comment:
This name may be slightly too generic for a trivial/sample server implementation.

Reply:
I agree. Do you have a suggestion for a better name? Maybe `serve-package` might be better

Reply:
Works for me.